File: LABORATÓRIO/Códigos feitos no Laboratório/Lab1RevisãoATP/Exercicio1JAVA/Programa.java
import java.util.Scanner;
class Programa{
    
    static double calculaVolume(double raio){
        double volume = 4*Math.PI*Math.pow(raio,3)/3;
        return volume;
    }
    public static void main(String[] args){
        Scanner entrada = new Scanner(System.in);
        System.out.println("Digite o valor do raio: ");
        double valor = entrada.nextDouble();
        System.out.println("O volume é: "+calculaVolume(valor));
    }
}

File: LABORATÓRIO/Códigos feitos no Laboratório/Lab1RevisãoATP/Exercicio1CSHARP/Programa.cs
using System;
class Programa{
    
    static double CalculaVolume(double raio){
        double volume = 4*Math.PI*Math.Pow(raio,3)/3;
        return volume;
    }
    public static void Main(string[] args){
        Console.WriteLine("Digite o valor do raio: ");
        Double.TryParse(Console.ReadLine(),out double valor);
        Console.WriteLine("O volume é: "+CalculaVolume(valor));
    }
}